en/core-libraries/toc-utilities.rst

```diff
@@ -1,22 +0,0 @@
-Utilities
-#########
```

Comment:
I have some concern that we'll be sending people who have bookmarks on the old pages to our ugly 404. That is a separate issue though.